docs/src/2q_and_edb_repositories.md

```diff
@@ -87,14 +87,14 @@ cluster_vars:
     - postgres_distributed
 ```
 
-This example will configure the `enterprise` and `postgres_distributed`
-repositories, giving access to EPAS and PGD5 products.
-On Debian or Ubuntu systems, it will use the APT repository, and on
-RedHat or SLES systems, it will use the rpm repositories, through the yum
-or zypper frontends, respectively.
+This example configures the `enterprise` and `postgres_distributed`
+repositories, giving access to EDB Postgres Advanced Server and PGD5 products.
```

Comment:
Is PGD5 a product separate from PGD? We don't refer to it as PGD5 if you mean PGD version 5.

Reply:
it's referring to PGD version 5

Reply:
```suggestion
repositories, giving access to EDB Postgres Advanced Server and PGD version 5 products.
```